skxray/img_proc/mathops.py

```diff
@@ -0,0 +1,77 @@
+# Module for the BNL image processing project
+# Developed at the NSLS-II, Brookhaven National Laboratory
+# Developed by Gabriel Iltis, Oct. 2013
+"""
+# This class is designed to facilitate image arithmetic and logical operations
+# on image data sets.
+"""
+
+import numpy as np
+
+
+def logical_nand(src_data1,
+                 src_data2):
+    """
+    This function enables the computation of the LOGICAL_NAND of two image or 
+    volume data sets. This function enables easy isolation of all data points 
+    NOT INCLUDED IN BOTH SOURCE DATA SETS. This function can be used for data 
+    comparison, material isolation, noise removal, or mask 
+    application/generation.
+
+    Parameters
+    ----------
+    src_data1 : array
+        Specifies the first reference data
+
+    src_data2 : array
+        Specifies the second reference data
+
+    Returns
+    -------
+    output : array
+        Returns the resulting array to the designated variable
+
+    Example
+    -------
+    result = mathops.logical_NAND('img_1', 'img_2')
+    """
+    output = np.logical_not(np.logical_and(src_data1, 
+                                           src_data2))
+    return output
+
+
+def logical_subtract(src_data1,

```

Comment:
is this equivalent to `A & !B` ? if it is false in A, it is False in the output, if it is True an A and False in B it is True in the output and if it is True in A and True in B it is False in the output

A  | B  | !B |  A & !B 
--- | ---- | --- | ---
T | T | F |F
T | F | T |T
F | F | T |F
F | T |  F |F